Regression/WarpX-tests.ini

```
@@ -1899,7 +1899,7 @@ analysisRoutine = Examples/analysis_default_regression.py
 [LaserAccelerationRZ_opmd]
 buildDir = .
 inputFile = Examples/Physics_applications/laser_acceleration/inputs_rz
-runtime_params = diag1.format=openpmd diag1.openpmd_backend=h5 max_step=20 diag1.fields_to_plot=Er Bt Bz jr jt jz rho part_per_cell part_per_grid rho_electrons
+runtime_params = diag1.format=openpmd max_step=20 diag1.fields_to_plot=Er Bt Bz jr jt jz rho part_per_cell part_per_grid rho_electrons
```

HDF5 is easier to install for testers and provided in CI:
```suggestion
runtime_params = diag1.format=openpmd diag1.openpmd_backend=h5 max_step=20 diag1.fields_to_plot=Er Bt Bz jr jt jz rho part_per_cell part_per_grid rho_electrons
```